lib/TzDataParser/Parser.php

```diff
@@ -4,77 +4,71 @@
 
 use Exception;
 
-class Parser {
-
-    public $rules = array();
-    public $zones = array();
+class Parser
+{
+    public $rules = [];
+    public $zones = [];
     protected $stream;
 
-    function parseFile($file) {
-
+    public function parseFile($file)
+    {
         $handle = fopen($file, 'r');
         if (!$handle) {
             throw new Exception("Could not open file: $file");
         }
         $this->stream = $handle;
         $this->parseStream();
-
     }
 
-    function parseStream() {
-
-        while($line = $this->getLine()) {
-
+    public function parseStream()
+    {
+        while ($line = $this->getLine()) {
             $this->parseLine($line);
-
         }
-
     }
 
-    protected function parseLine(array $line) { 
-
-        switch($line[0]) {
-            case 'Rule' :
+    protected function parseLine(array $line)
+    {
+        switch ($line[0]) {
+            case 'Rule':
                 $this->parseRule($line);
                 break;
-            case 'Zone' :
+            case 'Zone':
                 $this->parseZone($line);
                 break;
-            case 'Link' :
+            case 'Link':
                 break;
-            default :
-                echo "Unknown line: " . implode("\t", $line), "\n";
+            default:
+                echo 'Unknown line: '.implode("\t", $line), "\n";
                 break;
-
         }
-
     }
 
-    protected function parseRule(array $line) {
-
+    protected function parseRule(array $line)
+    {
         if (count($line) < 9) {
-            echo "Invalid rule: " . implode("--t--", $line);
-            break; 
+            echo 'Invalid rule: '.implode('--t--', $line);
+
+            return;
```

Comment:
Note: a local run of `phpstan` complained loudly about the `break` here. It should be something else - return false, throw an exception or???
I changed to to just `return` to at least allow `phpstan` to continue.